### PR TITLE
Dash image processing

### PR DESCRIPTION
**Pull request by YunkeXiao**
Issue for app: #35

# App pull request
- [ ] This is a new app
- [x] I am improving an existing app (redesigns/code "makeovers")

## About

- Playground deployment URL (new version): https://dash-playground.plotly.host/dash-image-processing/
- Current gallery app URL: https://dash-gallery.plotly.host/dash-image-processing/

## Workflow

- [x] I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [x] If the app is a redesigned and/or restyled version of an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.
- [x] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.
- [x] If the app is a reimplementation of a Python gallery app for the
  DashR gallery, the app in this PR mimics, as closely as possible,
  the style and functionality of the existing app.=
- [x] I have removed *all Google Analytics code* from the app's
  `assets/` folder.

## The pre-review review

I have addressed all of the following questions:

- [x] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.)
- [x] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)]
- [x] Am I reinventing the wheel? (I have used appropriate packages to
  lessen the volume of code that needs to be maintained.)

### CODE REVIEW

File: apps/dash-image-processing/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+dash==1.0.0
```
Comment (shammamah) [inline apps/dash-image-processing/requirements.txt]: Do we need all of these?

Comment (YunkeXiao) [reply apps/dash-image-processing/requirements.txt]: yes

File: apps/dash-image-processing/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+# Windows
+dash-image-processing-venv\Scripts\activate
+# Or Linux
+source venv/bin/activate
+```
+
+Clone the git repo, then install the requirements with pip
+```
+git clone https://github.com/plotly/dash-image-processing.git
```
Comment (celinehuang) [inline apps/dash-image-processing/README.md]: update to the monorepo link!

File: apps/dash-image-processing/README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+cd dash-image-processing
```
Comment (celinehuang) [inline apps/dash-image-processing/README.md]: cd dash-sample-apps/apps/dash-image-processing

File: apps/dash-image-processing/app.py [hunk 4]
```diff
@@ -0,0 +1,577 @@
...
+import pathlib
+import importlib
+
+import boto3
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import requests
+from dash.dependencies import Input, Output, State
+from dotenv import load_dotenv, find_dotenv
```
Comment (celinehuang) [inline apps/dash-image-processing/app.py]: remove unused imports!

File: apps/dash-image-processing/assets/custom_styles.css [hunk 5]
```diff
@@ -0,0 +1,464 @@
...
+        opacity: 0;
+    }
+    to {
+        opacity: 1;
+    }
+}
+
+/* Removes Undo Button
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+._dash-undo-redo {
```
Comment (celinehuang) [inline apps/dash-image-processing/assets/custom_styles.css]: remove this due to the new Dash update

**Pull request merged**